src/etapi/validators.ts

```diff
@@ -23,23 +23,23 @@ function isString(obj) {
     }
 }
 
-function isLocalDateTime(obj) {
-    if (obj === undefined || obj === null) {
+function isLocalDateTime(obj: ValidatorArg) {
+    if (obj === undefined || obj === null || typeof obj !== "string") {
```

Comment:
Nitpick: You could shorten this if-statement (and also the one on isUtcDateTime) to just 
`if (typeof obj !== "string")`

Reply:
Also, I don't know if I'm missing something here but: The type of obj (ValidatorArg) is `object | undefined | null`. So the if check will always be false, since you can't pass a string to this function?